pipeline/pipelines/gapic_generation_pipeline.py

```diff
@@ -104,9 +104,10 @@ def get_invalid_kwargs(self):
 class _PythonGapicTaskFactory(GapicTaskFactoryBase):
 
     def _get_gapic_package_tasks(self, **kwargs):
-        return [gapic_tasks.GapicCleanTask,
-                gapic_tasks.GapicCopyTask,
-                gapic_tasks.GapicPackmanTask]
+        return [
+            gapic_tasks.GapicCleanTask,
+            gapic_tasks.GapicCopyTask,
```

Comment:
Worth filing an issue to make this formatting consistent, and enforce it with Travis?